packages/application-generic/src/usecases/get-layout/get-layout.command.ts

```diff
@@ -1,7 +1,7 @@
 import { IsDefined, IsString } from 'class-validator';
-import { LayoutId } from '@novu/shared';
+import { LayoutId } from '@novu/dal';
```

Comment:
The reason of having the types imports from shared was to avoid have dependency in the DAL package and to be able to reuse the same typings across the monorepo and avoid some type mismatchings that happened because duplication in the different packages.

Reply:
I'll put it back